R/sparkR-mosaic/build_r_package.R

```diff
@@ -1,13 +1,15 @@
 repo<-"https://cran.ma.imperial.ac.uk/"
 
-install.packages("devtools", repos=repo)
+#install.packages("devtools", repos=repo)
```

Comment:
we should remove commented lines.

Reply:
done. also introduced the repo choosing logic